_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -88,7 +96,15 @@ VideoENCODE* _mfxSession::Create<VideoENCODE>(mfxVideoParam& par)
 
 #if defined(MFX_ENABLE_H265_VIDEO_ENCODE)
     case MFX_CODEC_HEVC:
-        pENCODE = new MfxHwH265Encode::MFXVideoENCODEH265_HW(&m_coreInt, &mfxRes);
+#if defined (MFX_ENABLE_HEVC_VIDEO_FEI_ENCODE)
+            feiEnabled = (bool*)core->QueryCoreInterface(MFXIFEIEnabled_GUID);
```

Comment:
I don't like idea of having pointer to bool as interface.

Reply:
As well as me, but this is what it is.